src/ui/components/Icon/styles.scss

```diff
@@ -10,10 +10,21 @@
   width: $default-icon-size;
 }
 
+.Icon-arrow {
+  @include icon($name: "arrow");
+}
+
+.Icon-arrow-blue {
+  @include icon($name: "arrow-blue");
+
+  [dir=rtl] & {
```

Comment:
should this also apply to `Icon-arrow`?

Reply:
It should; I think I do it in the component styles instead but it should really be done here. I'll update it.